fv3core/stencils/c2l_ord.py

```diff
@@ -3,7 +3,8 @@
 import fv3core._config as spec
```

Comment:
You could remove that by passing the namelist as an argument in `__init__`. The less we refer to globals the easiest it will get to clean them up later